prog/vm/github_runner.rb

```diff
@@ -43,7 +43,7 @@ def pick_vm
     end
 
     vm_st = Prog::Vm::Nexus.assemble_with_sshable(
-      "runner",
+      "runneradmin",
```

Comment:
We decided(I think) to standardize "ubi" as service user name.

Reply:
I think keeping it same with Github Runner's users is more important than keeping it standardized as ubi on our side. I don't want to cause a confusion on customer side by having different user information on github runners and our runners.

Reply:
I'm with @velioglu on this one, although the whole bifurcation based on runner size on Github is just strange

Reply:
> We decided(I think) to standardize "ubi" as service user name.

I agree with your point, but @velioglu also makes a valid argument. We're essentially playing by GitHub's rules here. I'm undecided. If standardizing as `ubi` eliminates some extra branches, it might have a chance.

Reply:
Ok, then I'm keeping it as it is, as Burak also approved the PR.